portfolio/src/main/java/com/google/sps/data/AUX.java

```diff
@@ -0,0 +1,97 @@
+package com.google.sps.data;
+
+import java.io.IOException;
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.model.VideoListResponse;
+import com.google.api.services.youtube.model.SearchListResponse;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+
+import java.io.StringWriter;
+import java.io.PrintWriter;
+
+import java.security.GeneralSecurityException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.ArrayList;
+
+import com.google.sps.data.Video;
+
+public class AUX{
+    private static String DEVELOPER_KEY;
+
+    private static final String APPLICATION_NAME = "Watch Wisely";
+    private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+
+    private static void GetDevKey(){
```

Comment:
I would recommend:
- getDevKey should be lower case, not uppercase.
- Rename it to maybeGetDevKey.
Gate this code if he DEVELOPER_KEY is null or empty, so 
if (DEVELOPER_KEY is null or empty) {
 // Load it from datastore.
} else {
 // Do nothing.
}

Reply:
Should the function be getdevkey(), GetDevKey() or getDevKey()?